lightstep/recorder.py

```diff
@@ -0,0 +1,94 @@
+from .basictracer.recorder import SpanRecorder
+from .crouton import ttypes
+
+import instrument
+import util
+import logging
+import constants
+import pprint
+import sys
+
+
+class Recorder(SpanRecorder):
+    """ SpanRecorder's job is record and report a BasicSpan.
+    """
+    def __init__(self, *args, **kwargs):
+        self.runtime = instrument.Runtime(*args, **kwargs)
```

Comment:
I think it might make sense to merge `Recorder` and `Runtime` into a single class (called `Recorder`). Otherwise this class is just a passthrough.